memory/memhotplug.py

```diff
@@ -195,7 +195,7 @@ def test_hotplug_toggle(self):
         self.__error_check()
 
     def test_dlpar_mem_hotplug(self):
-        if 'ppc' in platform.processor() and 'PowerNV' not in open('/proc/cpuinfo', 'r').read():
+        if 'ppc' in cpu.get_arch() and 'PowerNV' not in open('/proc/cpuinfo', 'r').read():
```

Comment:
I see the platform apis returning value correctly

```
>>> import platform
>>> platform.processor()
'ppc64le'
```

which distro has this problem ?

Reply:
@Naresh-ibm is this related https://github.com/avocado-framework/avocado/pull/5993

Reply:
This is python issue sir.
And I think the above util PR is different.